cmd/server/openapi/docs.go

```diff
@@ -907,15 +907,16 @@ const docTemplate = `{
                 }
             }
         },
-        "/org/lookup/{org_full_name}": {
+        "/orgs": {
             "get": {
+                "description": "Returns all registered orgs in the system. Requires admin rights.",
                 "produces": [
                     "application/json"
                 ],
                 "tags": [
-                    "Organizations"
+                    "Orgs"
```

Comment:
Unrelated, but what do you think about changing those tags from `Orgs` to `Organizations` for consistency in here as well.